aten/src/ATen/copy_wrapper.py

```diff
@@ -150,8 +150,13 @@ def create_one_copy(dst_type, all_types):
     else:
         function_fallthrough = FUNCTION_FALLTHROUGH_ERROR
 
+    checked_cast_dst = ''
+    if dst_type['Density'] == 'Dense':
```

Comment:
There is no longer a SparseFloatTensor class, so the checked cast no longer is valid.

But writing this made me realize that there's some nasty-pants code here, so I slimming it down some more.

Reply:
```
+    # Note [checked_cast_tensor is for dense only]
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # checked_cast_tensor is only needed for backends which implement
+    # copy and thus do a cast.  Sparse does not support copies, so there
+    # is no need to do a checked cast.  (Furthermore, the code as written
+    # will not work, as it will try to there is no derived Tensor type
+    # for sparse.)
```